### PR TITLE
Account banking partner journal account

### PR DESCRIPTION
**Pull request by ploegvde**
See: https://github.com/OCA/bank-payment/issues/121

Migration of module account_banking_partner_journal_account to V8.

### CODE REVIEW

File: account_banking_partner_journal_account/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+##############################################################################
+{
+    'name': 'Default partner journal accounts for bank transactions',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'author': "Therp BV,BAS Solutions,Odoo Community Association (OCA)",
+    'category': 'Banking',
+    'depends': ['base', 'account'],
+    'description': '''
+This module allows to set alternative journal accounts on partners to use
```
Comment (pedrobaeza) [inline account_banking_partner_journal_account/__openerp__.py:29]: Please extract this description to a README.rst file with OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: account_banking_partner_journal_account/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+#from openerp.osv import osv
```
Comment (pedrobaeza) [inline account_banking_partner_journal_account/res_partner.py]: Remove these lines.

File: account_banking_partner_journal_account/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        if not ids:
+            return {}
+        res = super(ResPartner, self).def_journal_account_bank_incr(
+            cr, uid, ids, context=context)
+        for partner in self.browse(cr, uid, ids, context=context):
+            if partner.property_account_receivable_bank_id:
+                res[partner.id] = partner.property_account_receivable_bank_id.id
+        return res
+        
+class account_bank_statement_line(orm.Model):
```
Comment (pedrobaeza) [inline account_banking_partner_journal_account/res_partner.py:70]: Use CamelCase and extract it to a file account_bank_statement.py.

**Comment by pedrobaeza** [comment]
Seeing again the module, maybe a better place is bank-import. What do you think?

**Comment by ploegvde** [comment]
Has nothing to do with import. It's sets a default account on reconciliation. This seems the right place to me.

**Comment by pedrobaeza** [comment]
Extracted from your description:

> This module allows to set alternative journal accounts on partners to use
> as default accounts in imported bank statements.

That was what leads me to think about it, but if you say it doesn't belong there, it's OK for me.

**Comment by ploegvde** [comment]
You are right. Text is from V7 situation. Changed it.

**Comment by ploegvde** [comment]
Just send CLA

**Comment by gurneyalex** [comment]
CLA checked ok.

**Comment by ploegvde** [comment]
Can you guys help me to get the module approved?

**Comment by pedrobaeza** [comment]
@ploegvde, I'm seeing that you closed this PR, but you're very closed to get the module merged. One of my comments is not honored, and that's why I haven't approved the PR yet.

**Pull request closed without merging**